client/components/remove-button/docs/example.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+* External dependencies
+*/
+var React = require( 'react' );
+
+/**
+ * Internal dependencies
+ */
+var RemoveButton = require( 'components/remove-button' ),
+	Card = require( 'components/card' );
+
+var RemoveButtons = React.createClass( {
+	displayName: 'RemoveButtons',
+
+	getInitialState: function() {
+		return {
+			compactButtons: false
+		};
+	},
+
+	render: function() {
+		var toggleButtonsText = this.state.compactButtons ? 'Normal Buttons' : 'Compact Buttons';
+
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/design/removebuttons">RemoveButton</a>
+					<a className="design-assets__toggle button" onClick={ this.toggleButtons }>{ toggleButtonsText }</a>
+				</h2>
+				{ this.renderButtons() }
+			</div>
+		);
+	},
+
+	renderButtons: function() {
```

Comment:
Formatting is off in this block as well. Also, note in the below screenshot that several of the lines are using spaces for indentation.

![screen shot](https://cloud.githubusercontent.com/assets/1126811/11490228/4442cf70-979c-11e5-9832-73aac68c3c8e.png)